web-client/public/js/generateNetwork.js

```diff
@@ -205,6 +205,12 @@ containing "-", "_", and alpha-numeric characters only`);
         $(CREATE_NETWORK_MODAL).modal("show");
     };
 
+    const sortGenes = function (genes, idToSort) {
```

Comment:
Solely for code readability, let’s define constants which assign a name to what the `0` and `1` IDs mean; something like:
```suggestion
    const GRN_STANDARD_NAME_ID = 1;
    const PPI_STANDARD_NAME_ID = 0;

    const sortGenes = function (genes, idToSort) {
```